### ISSUE TITLE
build with mlx5 failing

### ISSUE THREAD

**Issue by rishdas** [emmericp/MoonGen#217]
MoonGen/libmoon/deps/dpdk/drivers/net/mlx5/mlx5_rxtx.h:46:32: fatal error: infiniband/mlx5_hw.h: No such file or directory
compilation terminated.
/home/sirdas/code/common/MoonGen/libmoon/deps/dpdk/mk/internal/rte.compile-pre.mk:138: recipe for target 'mlx5.o' failed
make[4]: *** [mlx5.o] Error 1
/home/sirdas/code/common/MoonGen/libmoon/deps/dpdk/mk/rte.subdir.mk:63: recipe for target 'mlx5' failed
make[3]: *** [mlx5] Error 2
make[3]: *** Waiting for unfinished jobs....

The intial diagnosis says that the dpdk pointed by Moongen is old. Are there any plans of moving to a new dpdk ?
This is on a x86 machine
with MOFED libraries with version 4.2-1.2
and Ubuntu 16.04 Kernel 4.4 generic

**Comment by emmericp** [comment]
See readme for additional mlx requirements: https://github.com/libmoon/libmoon/blob/master/install-mlx.md

**Comment by vsag96** [comment]
@emmericp I installed the mellanox ofed driver and I am able to run the tespmd application of dpdk, how ever, I get the same error as above user. 

Mellanox driver does not install all the DEB's mentioned in the second option. I compared the output of Mellanox OFED install script with the deb's specified. I had to install libibverbs1, libibverbs-dev(not bs1-dev,there was no such deb) libmlx5, libmlx5-dev, mlnx-ofed-kernel-only. Rest were installed. Just putting it here in case, any one stumbles upon this. I can create a PR if you want. My ofed_info -s output was MLNX_OFED_LINUX-5.0-2.1.8.0.
